_data/lang/en/roadmap.yml

```diff
@@ -86,6 +86,15 @@
       status: completed
 - year: 2018
   accomplishments:
+    - name: New Proof of Work CryptoNoteV7
+      date: March, 2018
+      status: completed
+    - name: Network upgrade to increase minimal ringsize to 7, integrate multisig, subaddresses, and change PoW algo.
```

Comment:
Please remove the `dot .` at the end, it's no sentence. Also you can change the date of the item "New Proof of Work CryptoNoteV7" to the same date as the network upgrade.

Reply:
ok

Reply:
Btw i just saw i wrote 4/4 where the network upgrade occurs on the 6th